src/libasr/pass/pass_manager.h

```diff
@@ -37,7 +37,7 @@
 #include <libasr/pass/loop_vectorise.h>
 #include <libasr/pass/update_array_dim_intrinsic_calls.h>
 #include <libasr/pass/pass_array_by_data.h>
-
+#include <lpython/pickle.h>
```

Comment:
We can't include LPython nor LFortran in libasr. What we need some other way to do it. I think for now there might be a way to do this import with some ifdef preprocessor statements, but we really need to move the ASR printing into libasr itself from LFortran/LPython. That should be done in a separate PR.